packages/nc-gui/composables/useColumnCreateStore.ts

```diff
@@ -78,16 +78,16 @@ const [useProvideColumnCreateStore, useColumnCreateStore] = createInjectionState
       let columnName = `Untitled ${formState.value.uidt}`
       // Gets the no of columns in meta where meta.column_name === `Untitled${formState.value.uidt}`
       // Allows to count suffix differently for different UIDTs
-      let suffix = meta.value?.columns?.filter((c) => c.column_name?.includes(columnName)).length || 0
+      let columnNameSuffix = meta.value?.columns?.filter((c) => c.column_name?.includes(columnName)).length || 0
       while (
         (tableExplorerColumns?.value || meta.value?.columns)?.some(
           (c) => (c.column_name || '').toLowerCase() === columnName.toLowerCase(),
         )
       ) {
-        suffix++
-        columnName = `Untitled${suffix}`
+        columnNameSuffix++
+        columnName = `Untitled${columnNameSuffix}`
```

Comment:
1. double check if a space is needed between `Untitled` and `columnNameSuffix`
2. also should `formState.value.uidt` also be included?

Reply:
1. Yes I've added space columnNameSuffix in both places to get a more readable table name.
2. Also `formState.value.uidt` is required to generate different name for different UI types.
Passing it as an argument enables the table name to change on changing UITypes.

Reply:
What I meant is - you defined `columnName` as `Untitled ${formState.value.uidt}` in L78.  But in L88, you set `columnName` as `Untitled${columnNameSuffix}`. I was wondering if it should be `Untitled ${formState.value.uidt} ${columnNameSuffix}` ?